### PR TITLE
feat(src): add new structure to hero component

### PR DESCRIPTION
**Pull request by hanakoori01**
### What does this PR do ?

- Refactor hero component 

### Steps to test 
1. Run `yarn build`
2. Run `yarn start`
3. Go to `localhost:3000`
4. Go to Home

### CODE REVIEW

File: docs/src/pages/index.js [hunk 1]
```diff
@@ -1,45 +1,47 @@
...
-        </h1>
-        <div className={styles.buttons}>
-          <Link
-            className={clsx("button--lg", styles.buttonText)}
-            to="/docs/research/intro"
-          >
-            <Translate>Project Intro</Translate>
-          </Link>
+      <div className={styles.content}>
+        <div className={clsx("container", styles.container, )}>
```
Comment (tetogomez) [inline docs/src/pages/index.js:15]: Is the comma at the end of the second clsx params necessary?

File: docs/src/pages/index.js [hunk 2]
```diff
@@ -1,45 +1,47 @@
...
+            <Link
+              className={clsx("button--lg", styles.buttonText)}
+              to="/docs/research/intro"
+            >
+              <Translate>Project Intro</Translate>
+            </Link>
+          </div>
         </div>
+      </div>
+      <div className={clsx("container", styles.diagramContainer, )}>
```
Comment (tetogomez) [inline docs/src/pages/index.js:32]: same here!

**Review by tetogomez — CHANGES_REQUESTED** [review]
check the comments

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**